code/src/main/webapp/index.html

```diff
@@ -4,11 +4,25 @@
 <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <link rel="stylesheet" href="./CSS/style.css">
-   <title>First JSP</title>
+   <title>Welcome</title>
```

Comment:
Same as below - lets come up with a name for the site + then brand consistently.